Unicord.Universal/Unicord.Universal.csproj

```diff
@@ -646,486 +603,495 @@
     <Content Include="Assets\Store\Wide310x150Logo.scale-150.png" />
     <Content Include="Assets\Store\Wide310x150Logo.scale-200.png" />
     <Content Include="Assets\Store\Wide310x150Logo.scale-400.png" />
-    <None Include="Package.StoreAssociation.xml" />
     <Content Include="Controls\MarkdownTextBlock\About.txt" />
+    <None Include="Package.StoreAssociation.xml" />
     <Content Include="Properties\Default.rd.xml" />
   </ItemGroup>
   <ItemGroup>
-    <ApplicationDefinition Include="App.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </ApplicationDefinition>
-    <Page Include="Controls\AttachmentViewer.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </Page>
-    <Page Include="Controls\EmbedViewer.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\Embeds\EmbedFieldControl.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\Embeds\EmbedVideoControl.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </Page>
-    <Page Include="Controls\EmotePicker.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\Flyouts\AddServerFlyout.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\Flyouts\ChannelContextFlyout.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\Flyouts\DirectMessageContextFlyout.xaml">
+    <PackageReference Include="Humanizer">
```

Comment:
There's such a thing as central package management (not sure if it exists in UWP though) that allows you to have all your packages and dependencies in the root directory in `Dependencies.Packages.props` or a similar file - might be worth taking a look at if you're interested 😄

Reply:
Don't super see the point? Visual Studio has an okay UI for handling package versioning

Reply:
If you say so - though I think herbs NU1008 schema it is required in WinUI3. So I just thought it might be good to use going forward.